### ISSUE TITLE
unbale to install some binaries

### ISSUE THREAD

**Issue by muralikodali** [instl-sh/instl#2]
I am not able to install some binaries.

when i tried to install with `sudo instl euank/pazi, following is the output :  

```
     Running installer for github.com/euank/pazi


 SUCCESS  Found 2 assets in latest release!
 SUCCESS  Found an asset which seems to fit to your system:
┌────────────────────────────────────────────────────────────────────┐
| Name                               | Version | Last Update | Size  |
| pazi-x86_64-unknown-linux-musl.asc | v0.4.1  | 30 Oct 2019 | 833 B |
└────────────────────────────────────────────────────────────────────┘
 SUCCESS  Downloaded pazi-x86_64-unknown-linux-musl.asc
  FATAL   format unrecognized by filename: /usr/local/lib/instl/euank/pazi/pazi-x86_64-unknown-linux-musl.asc
panic:

goroutine 1 [running]:
github.com/pterm/pterm.checkFatal(...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:313
github.com/pterm/pterm.(*PrefixPrinter).Println(0xc0003b0100, 0xc0001bfa88, 0x1, 0x1, 0xc000492990)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:264 +0x1d9
github.com/pterm/pterm.PrefixPrinter.PrintOnError(0x9e004c, 0x7, 0xdc9960, 0x0, 0x0, 0x0, 0xdc9948, 0x1, 0xc0001bfc38, 0x1, ...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:209 +0x149
github.com/instl-sh/instl/cmd.glob..func3(0xdc9020, 0xc000113a50, 0x1, 0x1, 0x0, 0x0)
	/home/murali/Downloads/instl/cmd/root.go:162 +0x116c
github.com/spf13/cobra.(*Command).execute(0xdc9020, 0xc00013c010, 0x1, 0x1, 0xdc9020, 0xc00013c010)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xdc9020, 0x4013e0, 0x0, 0x0)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/instl-sh/instl/cmd.Execute()
	/home/murali/Downloads/instl/cmd/root.go:176 +0x2d
main.main()
	/home/murali/Downloads/instl/main.go:6 +0x25
```

for downloading alacritty terminal with `alacritty/alacritty` , i got the following error :  

```
  Running installer for github.com/alacritty/alacritty


 SUCCESS  Found 11 assets in latest release!
 SUCCESS  Found an asset which seems to fit to your system:
┌─────────────────────────────────────┐
| Name | Version | Last Update | Size |
|      |         | 01 Jan 0001 | 0 B  |
└─────────────────────────────────────┘
  FATAL   could not create download path: open /usr/local/lib/instl/alacritty/alacritty: is a directory
panic:

goroutine 1 [running]:
github.com/pterm/pterm.checkFatal(...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:313
github.com/pterm/pterm.(*PrefixPrinter).Println(0xc0005b5a40, 0xc0002e3a88, 0x1, 0x1, 0xc0000c3920)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:264 +0x1d9
github.com/pterm/pterm.PrefixPrinter.PrintOnError(0x9e004c, 0x7, 0xdc9960, 0x0, 0x0, 0x0, 0xdc9948, 0x1, 0xc0002e3c38, 0x1, ...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/pterm/pterm@v0.12.18/prefix_printer.go:209 +0x149
github.com/instl-sh/instl/cmd.glob..func3(0xdc9020, 0xc000099a50, 0x1, 0x1, 0x0, 0x0)
	/home/murali/Downloads/instl/cmd/root.go:158 +0x1026
github.com/spf13/cobra.(*Command).execute(0xdc9020, 0xc0000c2010, 0x1, 0x1, 0xdc9020, 0xc0000c2010)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xdc9020, 0x4013e0, 0x0, 0x0)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/murali/.gvm/pkgsets/go1.16.2/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/instl-sh/instl/cmd.Execute()
	/home/murali/Downloads/instl/cmd/root.go:176 +0x2d
main.main()
	/home/murali/Downloads/instl/main.go:6 +0x25
```

**Comment by MarvinJWendt** [comment]
Hi @muralikodali, thanks for the issue! Instl currently only supports releases, which contain a binary inside an archive, but I'll look into this, so that it can also install pure binary releases. This would fix Instl for `github.com/euank/pazi`.

For `github.com/alacritty/alacritty` it's different. They already push installers in their releases, and no compiled binaries. This can't be handled by Instl, as Instl itself is an installer, which is supposed to install binaries in GitHub releases. If the project pushes their own installers, it's probably better to use those anyway.

**Comment by MarvinJWendt** [comment]
Support for `github.com/euank/pazi` is fixed in `v1.7.0`
